Library/Homebrew/os/linux/elf.rb

```diff
@@ -68,6 +68,35 @@ def binary_executable?
     elf_type == :executable
   end
 
+  def runpath
+    return @runpath if defined? @runpath
+
+    @runpath = if HOMEBREW_PATCHELF_RB
```

Comment:
```suggestion
    @runpath ||= if HOMEBREW_PATCHELF_RB
```

Reply:
This suggestion does not work (does not cache the result) when the value is `nil`.

Reply:
```suggestion
    return @rpath if defined? @rpath

    @rpath = if HOMEBREW_PATCHELF_RB
```